hack/bump-kubevirt-provider.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+[[ -z "$1" ]] && echo "error: missing required tag argument" >&2 && exit 1
+TAG="$1"
+
+LATEST_K8S=$(curl -s "https://api.github.com/repos/kubevirt/kubevirtci/contents/cluster-up/cluster?ref=${TAG}" \
+  | grep '"name": "k8s-[0-9]\+\.[0-9]\+"' \
+  | sed -E 's/.*"k8s-([0-9]+\.[0-9]+)".*/\1/' \
+  | sort -V \
+  | tail -n1)
```

Comment:
Thanks
didn't look on all deeply yet but:

we cant take `tail -n1`
even if we wanted be on the bleeding edge (i.e k8s version higher than the one that is recommended at the very moment)
we can't, because when we freeze and create a branch, the branch need to use the recommended one

i.e release-0.97 uses k8s version X, not X+1 even if kubevirtci already has it

what you can do is to count, and always take the 3rd starting from oldest
on most of the cases it will work, it would stop working if the providers will have suddenly different naming,
as happened in the past, where we had
k8s-x-centos9 and such, which might breaks this formula, but we can do it meanwhile, and be proactive according needs

Reply:
ah I see. 
>what you can do is to count, and always take the 3rd starting from oldest

why third? I don't get the logic

Reply:
Sorry I don't get it.. How do we know which version is "the one we officially need to support"? can't we just use that?

Reply:
In the end we need to agree on the logic "which of the 3 will it choose"

Reply:
lets discuss offline and split to 2 PRs ?

Reply:
https://github.com/kubevirt/cluster-network-addons-operator/pull/2216#discussion_r2026834456

> Removing the KUBEVIRT_PROVIDER set. The nightly should use the default.

Food for thought: 
This can be an idea how to know the suggested provider to use,
we can suggest on kubevirtci, that it will know to use the latest by default, it will know the formula
and also will handle rare corner cases like we discussed